### ISSUE TITLE
feat: Ability to show codes in groups in the browser request view

### ISSUE THREAD

**Issue by LordSyd** [twofas/2fas-browser-extension#111]
### Feature description

I would love to have the option to have the codes in the browser request view to also be shown as groups, same as with the main view of the app.

### Motivation

Using the search feels slower to me than being able to collapse unneeded groups, scan the last open one visually, and click on the needed entry. In my case, I have a few websites with multiple users, so pairing the URL to a certain login is not possible.

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by GrzegorzZajac000** [comment]
Hi @LordSyd. Thank you for your report. Can you explain your problem further? We don't have any "browser request view" in browser extension. You can only request one token, from one service, and in browser you've got only TOTP token from that service.

Do you mean the service selection view in the mobile app?

**Comment by LordSyd** [comment]
Yeah, exactly, the service selection. I apologize for the confusion, I use the app in German and wasn't sure what it is called in English.

Is there something else I can or should clarify? :)

**Comment by GrzegorzZajac000** [comment]
Thank you @LordSyd. Everything is clear now, but this is a repository for extension, and the change you are writing about should be made in the mobile application. Are you using iOS or Android app?

**Comment by LordSyd** [comment]
Ah, sure, I can do that. I wasn't sure if it should go here or into the app repo. I will open a new request in the Android app repo in this case. Sorry for the inconvenience!

**Comment by GrzegorzZajac000** [comment]
There is no such need. I will transfer this issue ;)
